Parser/pgen/keywordgen.py

```diff
@@ -0,0 +1,60 @@
+"""Generate Lib/keyword.py from the Grammar and Tokens files using pgen"""
+
+import argparse
+
+from .pgen import ParserGenerator
+
+HEADER = '''\
+#! /usr/bin/env python3
+
+"""Keywords (from "graminit.c")
+
+This file is automatically generated; please don't muck it up!
+
+To update the symbols in this file, 'cd' to the top directory of
+the python source tree and run:
+
+    python -m Parser.pgen.keywordgen ./Grammar/Grammar \\
+                                     ./Grammar/Tokens \\
+                                     ./Lib/keyword.py
+
+Alternatively, you can run 'make regen-keyword'.
```

Comment:
Idea: rather than using a HEADER, why not using a template? Example: TEMPLATE = "xxx kwlist = [{kwlist}] xxx" and then render is using content = TEMPLATE.format(kwlist=...).

Reply:
Done in e2d2936!